vars/jira.groovy

```diff
@@ -1,35 +1,52 @@
 #!/usr/bin/env groovy
 
+def lastSuccessfulBuild(passedBuilds, build) {
+    if ((build != null) && (build.result != 'SUCCESS')) {
+        echo "Finding last succesful build"
+        passedBuilds.add(build)
+        echo "Found a build"
+        lastSuccessfulBuild(passedBuilds, build.getPreviousBuild())
+    }
+}
+
```

Comment:
where is this function being used. Can it be rewritten as a simple filter closure?

Reply:
https://github.com/cliqz-oss/cliqz-android/blob/6a19ca2b9dc484cba14f6fbbba90a4bc2aa85d6f/Jenkinsfile.nightly#L149 is an example of where the function is being used. Not sure how it would work as a closure.